.github/workflows/nightly_ci.yaml

```diff
@@ -25,13 +25,14 @@ jobs:
       with:
         use-quiet-mode: 'yes'
         folder-path: tutorials-v${{ matrix.qutip-version }}
+        config-file: mlc_config.json
```

Comment:
Maybe we could add a quick comment what this is good for and why it was necessary?